namepsace stragegies: Updated (#2049)
codenvy/che-docs: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc

[id="configuring-namespace-strategies_{context}"]
= Configuring workspace target {orch-namespace}

> The {platforms-namespace} where a new workspace is deployed depends on the {prod-short} server configuration. {prod-short} deploys each workspace in to a user's dedicated {orch-namespace}, which hosts all {prod-short} workspaces created by the user. The name of a {platforms-namespace} must be provided as a {prod-short} server configuration property or pre-created by {prod-short} administrator.

Reviewer: ```suggestion
The {platforms-namespace} where a new workspace is deployed depends on the {prod-short} server configuration. {prod-short} deploys each workspace into a user's dedicated {orch-namespace}, which hosts all {prod-short} workspaces created by the user. The name of a {platforms-namespace} must be provided as a {prod-short} server configuration property or pre-created by {prod-short} administrator.
```
